fe/fe-core/src/main/java/com/starrocks/catalog/Replica.java

```diff
@@ -301,6 +301,11 @@ public synchronized void updateVersionInfo(long newVersion,
                 lastSuccessVersion, dataSize, rowCount);
     }
 
+    public synchronized void updateVersion(long version) {
+        updateReplicaInfo(version, this.lastFailedVersion,
```

Comment:
What's the meaning of the lastFailedVersion and lastSuccessVersion after refactor?

Reply:
lastFailedVersion will be set if version is left behind
I don't change lastSuccessVersion's logic, so it's same as before